lang/test/org/partiql/lang/TestBase.kt

```diff
@@ -73,10 +71,10 @@ abstract class TestBase : Assert() {
     }
 
 
-    protected fun assertBaseRewrite(originalSql: String, exprNode: ExprNode) {
-        val clonedAst = defaultRewriter.rewriteExprNode(exprNode)
+    protected fun assertBaseTransform(originalSql: String, exprNode: ExprNode) {
+        val clonedAst = defaultTransform.transformStatement(exprNode.toAstStatement()).toExprNode(ion)
         assertEquals(
-            "AST returned from default AstRewriterBase should match the original AST. SQL was: $originalSql",
+            "AST returned from default PartiqlAst.VisitorTransform should match the original AST. SQL was: $originalSql",
             exprNode, clonedAst)
     }
```

Comment:
I changed the call from `rewriteExprNode` to `transformStatement`. To get this working, I had to fix some of the passing of metas in [`ExprNodeToStatement`](https://github.com/partiql/partiql-lang-kotlin/pull/342/files#diff-adbd0081150155dc39c0df60f859fa7cf962e821a67b78395993ce837424e0ad). An alternative to comparing `ExprNode`s could be to compare `PartiqlAst`s i.e.:
```kotlin
protected fun assertBaseTransform(originalSql: String, exprNode: ExprNode) {
    val partiqlAst = exprNode.toAstStatement()
    val clonedAst = defaultTransform.transformStatement(partiqlAst)
    assertEquals("...", partiqlAst, clonedAst)
}
```
Should I be comparing `ExprNode`s or `PartiqlAst`s here?

Reply:
Yeah, this function is intended to test `AstRewriterBase` and should continue to test it.  This change should probably be undone before you merge this.

For the PIG-generated equivalent its not necessary since we can assume that it functions correctly.  (PIG itself should have a test like this.  If it doesn't we should create one.)

Reply:
Got it. Reverted the function back to test `AstRewriterBase`